src/components/theme-card.tsx

```diff
@@ -22,19 +22,21 @@ export default function ThemeCard({
 	const maxDescLen = 100;
 	const authorLink = getThemeAuthorLink(theme);
 
+	const handleMouseDown = (e: React.MouseEvent) => {
+		if (e.button !== 0 && e.button !== 1) return;
+		if (e.target instanceof HTMLAnchorElement) return;
+		window.open(`/mods/${theme.id}`, e.button === 1 ? "_blank" : "_self");
+	};
+
 	return (
 		<Card
-			className="h-full select-none flex-col justify-between rounded-xl border-2 border-[transparent] bg-surface transition-all duration-200 hover:border-[rgba(0,0,0,.5)] hover:shadow-lg dark:bg-[#121212] dark:hover:border-[#333]"
-			onMouseDown={(e) => {
-				// IMPORTANT NOTE: We do NOT use a Link component here because of how zen manages site injection.
-				//   please for the love of god, dont change this to a Link component. Please.
-				//
-				// If you had the tentation to change this to a Link component and saw this comment, please
-				//   increase this number: 1
-				if (e.button !== 0 && e.button !== 1) return;
-				if (e.target instanceof HTMLAnchorElement) return;
-				window.open(`/mods/${theme.id}`, e.button === 1 ? "_blank" : "_self");
-			}}
+			className="h-full cursor-pointer select-none flex-col justify-between rounded-xl border-2 border-[transparent] bg-surface transition-all duration-200 hover:border-[rgba(0,0,0,.5)] hover:shadow-lg dark:bg-[#121212] dark:hover:border-[#333]"
+			onMouseDown={handleMouseDown}
```

Comment:
Please move the comment to inside the function

Reply:
@mauro-balades  here? 
![image](https://github.com/user-attachments/assets/5cfb87ab-97cd-44ea-8c22-06dd78a20f7f)

Reply:
on top

Reply:
in the top of const or top top of the function itself
![image](https://github.com/user-attachments/assets/5daa4a00-6409-43fa-9063-42285bca37ed)

Reply:
good now? 
![image](https://github.com/user-attachments/assets/b53fd20e-e69a-472f-b5eb-eb36fe405b82)